lib/pf/UnifiedApi/Controller/Violations.pm

```diff
@@ -0,0 +1,80 @@
+package pf::UnifiedApi::Controller::Violations;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Violations -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Violations
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::violation;
+
+has dal => 'pf::dal::violation';
+has id_key => 'id';
+has resource_id => 'id';
+
+
+sub list {
```

Comment:
@satkunas 

Not sure why listing violations would only list the open ones.
This should list all the violations and then we could have /violations/opened to list the opened violations

What do you think?

Reply:
I didn't see an existing method in the base class that returned all (including non-open) violations. Please correct me if I'm wrong.

Reply:
If you just go through the DAL, that will perform an unfiltered select

@jrouzierinverse can confirm this.

So basically, you'd just have to delete this subroutine and we'd be golden (again @jrouzierinverse  can confirm)